packages/workspace-resolver/src/resolve-workspace.ts

```diff
@@ -62,13 +62,14 @@ export async function resolveWorkspace(
 > {
   const workingDirToUse = workingDir ?? process.cwd();
   const isRoot = workingDirToUse === root;
-  const path = packageJsonPath(root);
-  const json = await readPackageJson(isRoot, workingDirToUse, path);
+  const pkgPath = packageJsonPath(root);
```

Comment:
Renamed `path` => `pkgPath` because `path` conflicts with the internal node module